devicesetup/build.gradle

```diff
@@ -1,10 +1,11 @@
 apply plugin: 'com.android.library'
 apply plugin: 'me.tatarka.retrolambda'
 
-
 // This is the library version used when deploying the artifact
 version = '0.4.0'
 
+def analytics = "false"
```

Comment:
Are you planning to just change these values manually on your local build and then not commit the changes?